### PR TITLE
Add paper-button with the ability to delete greetings.

### PR DESCRIPTION
**Pull request by YuriiHaidamaka**
In this PR we have added the following functionality:
 - Each item in the list has the button which removes this item from the list.

### CODE REVIEW

File: src/dynamic-array-app/dynamic-array-app.html [hunk 1]
```diff
@@ -47,6 +51,11 @@
                 this.addName(this.inputValue);
             }
 
+            deleteName(e) {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: There is a convention that if you want to declare a protected method it should have `_` in the beginning: `_deleteName`. If private — should be `__`.

**Comment by YuriiHaidamaka** [comment]
I have added "_" in protected methods and have deleted bower components directory.
@dkalinin PTAL

File: src/dynamic-array-app/dynamic-array-app.html [hunk 2]
```diff
@@ -48,15 +48,19 @@
             }
 
             handleClick() {
-                this.addName(this.inputValue);
+                this._addName(this.inputValue);
             }
 
-            deleteName(e) {
-                var index = e.model.index;
+            handleDeleteClick(e) {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: This should be private too.

File: src/dynamic-array-app/dynamic-array-app.html [hunk 3]
```diff
@@ -44,10 +48,19 @@
             }
 
             handleClick() {
-                this.addName(this.inputValue);
+                this._addName(this.inputValue);
+            }
+
+            handleDeleteClick(e) {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: Event handlers are usually private.

File: src/dynamic-array-app/dynamic-array-app.html [hunk 4]
```diff
@@ -44,10 +48,19 @@
             }
 
             handleClick() {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: Event handlers are usually private.

**Comment by YuriiHaidamaka** [comment]
@dkalinin PTAL

**Pull request merged**